cmake/f3dPlugin.cmake

```diff
@@ -69,22 +69,42 @@ macro(f3d_plugin_declare_reader)
     set(F3D_READER_HAS_SCORE 0)
   endif()
 
-  set(F3D_READER_INCLUDES_CODE
-    "${F3D_READER_INCLUDES_CODE}#include \"reader_${F3D_READER_NAME}.h\"\n")
-  set(F3D_READER_REGISTER_CODE
-    "${F3D_READER_REGISTER_CODE}std::make_shared<reader_${F3D_READER_NAME}>(),")
+  set(F3D_PLUGIN_INCLUDES_CODE
+    "${F3D_PLUGIN_INCLUDES_CODE}#include \"reader_${F3D_READER_NAME}.h\"\n")
+  set(F3D_PLUGIN_REGISTER_CODE
+    "${F3D_PLUGIN_REGISTER_CODE}std::make_shared<reader_${F3D_READER_NAME}>(),")
 
+  # TODO: remove this and read JSON in shellext
```

Comment:
do you plan to do it or is it a todo for the future ?

Reply:
would be nice to do as a way of testing the actual feature

Reply:
it's done :)